Distraction Shield/background.js

```diff
@@ -92,12 +97,23 @@ function intercept(details) {
 }
 
 /**
+ * This function checks if a string url is whitelisted.
+ * @see {module:constants.whitelist}
+ * @param {String} url The string to check against the regexp's in whitelist
+ * @returns {Boolean} True if url matches one of the regexp's in whitelist, false otherwise
+ */
+function isWhiteListed(url) {
+    let matches = constants.whitelist.map((x) => { return x.test(url) });
+    return matches.reduce((x, y) => { return x || y });
+}
```

Comment:
it doesn't seem appropriate to me to put this function in the background. maybe put it in the stringutil module? and pass it the 2 arguments (whitelist, url) to make it more general.